src/Kernel/CompiledMethod.class.st

```diff
@@ -1154,6 +1154,11 @@ CompiledMethod >> usesUndeclareds [
 	^ false
 ]
 
+{ #category : #evaluating }
+CompiledMethod >> valueWithReceiver: aReceiver [
+	^self receiver: aReceiver withArguments: #() executeMethod: self
```

Comment:
Aren't too much synonym for the same thing?
In fact, I did not even realize that `CompiledMethod>>#receiver:withArguments:executeMethod:` existed.
(and why the compiled method is both a receiver and an argument here?)

Up to now, I used `ProtoObject>>#executeMethod:`. That one uses `ProtoObject>>#withArgs:executeMethod:` that is the same primitive number 188. Weird.

Reply:
I added a comment in #receiver:withArguments:executeMethod: